snp_blending/launch/start.launch.xml

```diff
@@ -88,9 +99,9 @@
   </include>
 
   <!-- Launch Motion Execution Node -->
-  <node pkg="snp_motion_execution" exec="motion_execution_server">
+<!--  <node pkg="snp_motion_execution" exec="motion_execution_server">
     <param name="follow_joint_trajectory_action" value="$(var follow_joint_trajectory_action)"/>
-  </node>
+  </node>-->
```

Comment:
Is this now a BT in the GUI?

Reply:
I see it's still getting launched in the snp_automate_2022 `start.launch`

Reply:
It should be removed in both. The BT now does all of the steps that this service node was doing. It probably makes sense to just get rid of this node now